auth/src/main/java/com/firebase/ui/auth/data/model/FlowParameters.java

```diff
@@ -128,4 +131,24 @@ public FlowParameters createFromParcel(Parcel in) {
             return new FlowParameters[size];
         }
     };
+
+    public boolean isSingleProviderFlow() {
+        return providerInfo.size() == 1;
+    }
+
+    public @StringRes int getGlobalTermsStringResource() {
+        boolean hasTos = !TextUtils.isEmpty(termsOfServiceUrl);
+        boolean hasPp = !TextUtils.isEmpty(privacyPolicyUrl);
+        if(hasTos && hasPp) {
+            return R.string.fui_tos_and_pp;
+        } else if(hasTos) {
+            return R.string.fui_tos_only;
+        } else {
+            return R.string.fui_pp_only;
```

Comment:
This else implies the default is the privacy policy—is that what you want? If so, we should add checks, update the README n' docs, throw exceptions, yada yada for `AuthUI` to ensure the developer must supply either a tos or a pp. Otherwise, we should return -1 and handle that case.

Reply:
You're right, I'll change this. We won't force them to have a ToS/Privacy policy though.